components/expandable-section/src/expandable-section.ts

```diff
@@ -14,29 +14,27 @@
  * limitations under the License.
  */
 
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
 import {
   ChangeDetectionStrategy,
   ChangeDetectorRef,
   Component,
+  Directive,
+  EventEmitter,
   Input,
-  NgZone,
+  OnChanges,
   Output,
+  SimpleChanges,
   ViewChild,
   ViewEncapsulation,
 } from '@angular/core';
-import { Observable, defer } from 'rxjs';
-import { filter, switchMap, take } from 'rxjs/operators';
-
 import { CanDisable } from '@dynatrace/barista-components/core';
 import { DtExpandablePanel } from '@dynatrace/barista-components/expandable-panel';
+import { filter } from 'rxjs/operators';
 
-@Component({
+@Directive({
   exportAs: 'dtExpandableSectionHeader',
   selector: 'dt-expandable-section-header',
-  template: '<ng-content></ng-content>',
-  encapsulation: ViewEncapsulation.Emulated,
-  preserveWhitespaces: false,
-  changeDetection: ChangeDetectionStrategy.OnPush,
 })
```

Comment:
Can we just change this from a component to a Directive? This immensely breaks behaviour, as there is no more `ng-content` outlet.

Reply:
Imo this should be fine - you can add any content inside a directive :)

Reply:
we just used it for selecting the content. The `ng-content` was useless. Same public API after the changes as before.

![image](https://user-images.githubusercontent.com/11156362/75145129-aa769e00-56f8-11ea-8e4a-5b746d46049f.png)